packages/backend/src/open_payments/auth/middleware.ts

```diff
@@ -37,6 +118,17 @@ export function createAuthMiddleware({
       if (!grant || !grant.active) {
         ctx.throw(401, 'Invalid Token')
       }
+      if (!config.skipSignatureVerification) {
```

Comment:
auth middleware tests should be updated to test signature verfication:
https://github.com/interledger/rafiki/blob/3d30ce755b4218a777ec77f408be5d15dfa0e687/packages/backend/src/open_payments/auth/middleware.test.ts#L76
(which would require making `skipSignatureVerification` false just for that test file ?)

Reply:
Tests have been added